examples/with-google-analytics/pages/_document.js

```diff
@@ -0,0 +1,30 @@
+import { Html, Head, Main, NextScript } from 'next/document'
+import * as gtag from '../lib/gtag'
+
+export default function Document() {
+  return (
+    <Html>
+      <Head />
+      <body>
+        <Main />
+        <NextScript />
+
+        {/* Global Site Tag (gtag.js) - Google Analytics */}
+        <script
+          async
+          src={`https://www.googletagmanager.com/gtag/js?id=${gtag.GA_TRACKING_ID}`}
+        />
```

Comment:
`https://www.googletagmanager.com/gtag/js` is also loaded in the `_app` aready. We only need to inline `window.dataLayer` and `function gtag(){}` in the `_document`.

```suggestion
```